modules/shell/command.go

```diff
@@ -17,10 +17,11 @@ import (
 
 // Command is a simpler struct for defining commands than Go's built-in Cmd.
 type Command struct {
-	Command    string            // The command to run
-	Args       []string          // The args to pass to the command
-	WorkingDir string            // The working directory
-	Env        map[string]string // Additional environment variables to set
+	Command           string            // The command to run
+	Args              []string          // The args to pass to the command
+	WorkingDir        string            // The working directory
+	Env               map[string]string // Additional environment variables to set
+	OutputMaxLineSize *int              // The max line size of stdout and stderr (in bytes)
```

Comment:
Does this need to be a pointer? Or can we get away with `int`, and the default behavior treats `0` as a special case?

Reply:
Hi, thanks for the information. I will provide an update soon.